main.tf

```diff
@@ -48,6 +48,7 @@ module "vpc" {
   private_subnet_tags = {
     Name                                        = "${var.environment}-${var.cluster_name}-private"
     "kubernetes.io/cluster/${var.cluster_name}" = "shared"
+    "kubernetes.io/role/internal-elb"           = ""
```

Comment:
Key: kubernetes.io/role/internal-elb
Value: 1 
This tag is required if you want to use not all networks, only those that have this tag. It is necessary only if your architecture includes dedicating instances for ingress or some specific case. By default, we don't need to add this, but a good point to get the ability to set this if required. Maybe add some variables for this?